index.md

```diff
@@ -15,47 +14,38 @@ type: documentation
-
-So, from now on, only a GitHub account is needed to contribute :)
+If you wish to contribute to the cheat sheets, or to suggest any improvements or changes, then please do so via the issue tracker on the [GitHub](https://github.com/OWASP/CheatSheetSeries) repository. Alternatively, join us in the `#cheetsheats` channel on the OWASP Slack (details in the sidebar).
 
 ## Bridge between the projects OWASP Proactive Controls, OWASP ASVS, and OWASP CSS
 
-A work channel has been created between OWASP Proactive Controls (OPC), OWASP Application Security Verification Standard (OASVS), and OWASP Cheat Sheet Series (OCSS) using the following process:
+A work channel has been created between OWASP Proactive Controls (OPC), OWASP Application Security Verification Standard (ASVS), and OWASP Cheat Sheet Series (OCSS) using the following process:
 
-- When a Cheat Sheet is missing for a point in OPC/OASVS, then the OCSS will handle the missing and create one. When the Cheat Sheet is ready, then the reference is added by OPC/OASVS.
-- If a Cheat Sheet exists for an OPC/OASVS point but the content do not provide the expected help then the Cheat Sheet is updated to provide the required content.
+- When a Cheat Sheet is missing for a point in OPC/ASVS, then the OCSS will handle the missing and create one. When the Cheat Sheet is ready, then the reference is added by OPC/ASVS.
+- If a Cheat Sheet exists for an OPC/ASVS point but the content do not provide the expected help then the Cheat Sheet is updated to provide the required content.
 
-The reason of the creation of this bridge is to help OCSS and OASVS projects by providing them:
+The reason of the creation of this bridge is to help OCSS and ASVS projects by providing them:
 
 - A consistent source for the requests regarding new Cheat Sheets.
 - A shared approach for updating existing Cheat Sheets.
 - A usage context for the Cheat Sheet and a quick source of feedback about the quality and the efficiency of the Cheat Sheet.
 
-It is not mandatory that a request for a new Cheat Sheet (or for an update) comes only from OPC/OASVS, it is just an extra channel.
+It is not mandatory that a request for a new Cheat Sheet (or for an update) comes only from OPC/ASVS, it is just an extra channel.
+
+> Requests from OPC/ASVS are flagged with a special label in the GitHub repository issues list in order to identify them and set them as a top level priority.
+
+## Project Leaders
 
-> Requests from OPC/OASVS are flagged with a special label in the GitHub repository issues list in order to identify them and set them as a top level priority.
+- [Jim Manico](mailto:jim.manico@owasp.org)
 
 ## Core Team
 
-- [Jakub Maćkowski](https://github.com/mackowski)
-- [Robin Bailey](https://github.com/rbsec/)
+- [Elie Saad](mailto:eliesaad7@gmail.com)
```

Comment:
Move Elie to **Project Leaders** section

Reply:
Fixed